src/components/tooltip/index.tsx

```diff
@@ -0,0 +1,169 @@
+import {
```

Comment:
Inspired by the 'reusable' component here: https://floating-ui.com/docs/tooltip#examples